lib/netsuite/configuration.rb

```diff
@@ -15,6 +15,8 @@ def attributes
     end
 
     def connection(params={}, credentials={})
+      proxy_params = proxy ? {proxy: proxy} : {}
```

Comment:
@dbecker-stripe  Can you add a test for setting the proxy config to the test suite?

Reply:
Added testing. I'm not sure they will catch the bug given that it requires an older version of savon but the tests run against the latest version. If there is a change in savon behavior these tests will catch it.